README.md

```diff
@@ -1,18 +1,18 @@
-# SD-Core NSSF Operator (k8s)
-[![CharmHub Badge](https://charmhub.io/sdcore-nssf/badge.svg)](https://charmhub.io/sdcore-nssf)
+# SD-Core NSSF K8s Operator
+[![CharmHub Badge](https://charmhub.io/sdcore-nssf-k8s/badge.svg)](https://charmhub.io/sdcore-nssf-k8s)
 
-Charmed Operator for the SD-Core Network Slice Selection Function (NSSF).
+Charmed K8s Operator for the SD-Core Network Slice Selection Function (NSSF).
```

Comment:
`Charmed K8s Operator` to me, sounds a bit like it was meant only for Charmed K8s. If it's just my opinion, feel free to ignore this ;)